src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -132,6 +132,12 @@ class StateResponse(str, Enum):
     SUCCEEDED = "SUCCEEDED"
     TERMINATED = "TERMINATED"
     FAILED = "FAILED"
+    KILLED = "KILLED"
+    UNKNOWN = "UNKNOWN"
+
+    @classmethod
+    def _missing_(cls, _):
+        return cls.UNKNOWN
```

Comment:
**praise**: thanks for making it open-ended! TIL: [the enum `_missing_` hook](https://docs.python.org/3/library/enum.html#enum.Enum._missing_)